assets/javascripts/initializers/discourse-calendar.js.es6

```diff
@@ -434,30 +434,43 @@ function initializeDiscourseCalendar(api) {
     calendar.addEvent(event);
   }
 
-  function _setDynamicCalendarEvents(calendar, post, siteSettings) {
+  function _setDynamicCalendarEvents(calendar, $calendar, post, siteSettings) {
```

Comment:
Could we avoid using jQuery here by passing in the fullDay as a parameter instead of the jQuery reference?